go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/Azure/azure-sdk-for-go v37.1.0+incompatible
 	github.com/Azure/azure-storage-blob-go v0.8.0
 	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
-	github.com/Azure/go-autorest/autorest v0.9.6
+	github.com/Azure/go-autorest/autorest v0.11.12
```

Comment:
@narph Could it be a problem if merged before https://github.com/elastic/beats/pull/28028. I can wait for your PR to merged first if needed.